src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -879,6 +891,7 @@ void VM_Version::get_processor_features() {
 
   if (cpu_family() > 4) { // it supports CPUID
     _features = _cpuid_info.feature_flags(); // These can be changed by VM settings
+    _extra_features = _cpuid_info.extra_feature_flags(); // These can be changed by VM settings
     _cpu_features = _features;   // Preserve features
```

Comment:
`_cpu_features` now captures only part of CPU features JVM cares about.

Reply:
It has limited usage and was introduced by JDK-8324734, apparently, cthe omment over it mentions
+  // Feature identification not affected by VM flags
https://github.com/openjdk/jdk/pull/17590#issuecomment-1912689152

Reply:
I don't see how JVMCI is relevant here. 

JDK-8324734 was related to AVX512. I don't see why a very similar scenario won't be applicable to AVX10.

Reply:
Yes, its has a limited usage only in C2 context.